app/controllers/api/v1/inventory_numbers_controller.rb

```diff
@@ -19,7 +19,9 @@ def create
 
       api :PUT, "/v1/inventory_numbers", "Delete inventory_number"
       def remove_number
-        InventoryNumber.find_by(code: params[:code]).destroy
+        if (inventory_number = InventoryNumber.find_by(code: params[:code]))
+          inventory_number.destroy
+        end
```

Comment:
One liner alternative: `InventoryNumber.find_by(code: params[:code])&.destroy`

Reply:
@patrixr , I was going to comment the same thing. Ruby provides safe navigation operator while rails provides `try` method from ActiveSupport. 

I would like to know which one we should prefer. 
@swatijadhav and @namrataukirde I would love your input here.

I referred: https://stackoverflow.com/questions/45825363/what-is-the-difference-between-try-and-safe-navigation-operator-in-ruby

Reply:
As mentioned in the reference, both are valid based on the use case. 
But I would prefer to go with `.try` as it won't raise any unnecessary exception, also throughout the app it would be consistent to use and there won't be confusion.

Reply:
Thanks @swatijadhav  👍

Reply:
I agree with @swatijadhav , but my reasons for preferring `try` is that it's explicit and descriptive as to what it does. Anyone reading it immediately understands that the record might not exist. The `&` kinda goes unnoticed.

Reply:
Also, I have absolutely nothing a good old `if (exists) { destroy }`. Easy to understand, clear, efficient.
--> Less code is not better code

Reply:
Thanks, everyone for the input. Nice discussion. 

I missed using try instead of if. In my opinion, I will prefer try over safe navigation for both the same reasons which @swatijadhav mentioned.  

Also, it looks clearer to me. (It's my personal preference over safe navigation 😅)

I will do suggested change and will let you guys know.